Services/CRUD/CrudTransformer.php

```diff
@@ -77,10 +80,60 @@ public function __construct(
     }
 
     /**
-     * Array to collection transform
+     * Array to object transformation
      *
-     * @param array  $data  Data
-     * @param string $class Given collection class
+     * @param array  $data  Object fields
+     * @param string $class Class name of object to create
+     * @return mixed
+     * @throws ConstraintViolation
+     */
+    public function arrayToObject(array $data, $class)
+    {
+        $this->initializeClassMetadata($class);
+        $object = new $class();
+        foreach ($data as $property => $value) {
+            $this->processPropertyValue(
+                $object,
+                $property,
+                $value,
+                'create'
+            );
+        }
+        return $object;
+    }
+
+    /**
+     * Array to object property validation
+     *
+     * @param array  $data  Object fields
+     * @param string $class Class name of object to test
+     * @return ConstraintViolationList
+     */
+    public function arrayToObjectPropertyValidation(array $data, $class)
+    {
+        $badProperties = new ConstraintViolationList();
+        foreach ($data as $property => $value) {
+            if (!$this->isPropertyAccessible($property, 'create')) {
```

Comment:
@RuslanZavacky This is handling absent property validation. You mentioned you wanted to think about the use of this.